### ISSUE TITLE
Option to disable context menu

### ISSUE THREAD

**Issue by JonnieCache** [equinusocio/material-theme#1055]
Can we have the ability to disable the context menu entry please? Or just get rid of it altogether? Configuring the theme is something that one does only once, (and incidentally I'd  rather do it via the json file anyway, but that's a personal preference.)

It's redundant to have a context menu entry there every time I right click in the main editor panel when it's identical to the one in the preferences menu, which is the proper place for it anyway IMO. Configuration of the theme is not part of the "context" of the document editor - one expects to find operations relating specifically to the thing one has right-clicked on.

This may seem trivial but if you're on a 13 inch laptop then every pixel counts. I suppose I should just force myself to always use the command palette really but I seem to have muscle memory for the context menu regarding certain tasks - particularly ones which require the mouse anyway, eg. show in finder.

(I've tried extracting the package and deleting the `Context.sublime-menu` file, but it doesn't work. I've never quite got the hang of the load-order precedence for sublime packages - it doesnt seem to function as expected.)

The theme is lovely BTW!

**Comment by equinusocio** [comment]
Open a poll issue :)

**Comment by equinusocio** [comment]
Open a poll issue :). 

If you want to force it, just clear the Context.sublime-menu and leave it empty.
